subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -459,10 +459,6 @@ uint8_t ull_cp_chan_map_update(struct ll_conn *conn, uint8_t chm[5])
 {
 	struct proc_ctx *ctx;
 
-	if (conn->lll.role != BT_HCI_ROLE_MASTER) {
-		return BT_HCI_ERR_CMD_DISALLOWED;
-	}
-
```

Comment:
Think this makes test_channel_map_update_sla_loc fail in tests/bluetooth/controller/ctrl_chmu/bluetooth.controller.ctrl_chmu.test

Reply:
yes